doc/concepts/concepts.rst

```diff
@@ -1,13 +1,227 @@
+========
 Concepts
 ========
 
-These are the **why-docs** that discuss the design of MoveIt and this website.
+The following list major MoveIt concepts and components and describes how they tie in with each other. For tutorials, see the `tutorials <https://moveit.picknik.ai/>`_ page.
+
+.. image:: /_static/images/moveit_pipeline.png
+
+***********************
+The ``move_group`` node
+***********************
+
+The figure above shows the high-level system architecture for the primary node provided by MoveIt called `move_group <http://docs.ros.org/noetic/api/moveit_ros_move_group/html/move__group_8cpp.html>`_. This node serves as an integrator: pulling all the individual components together to provide a set of ROS actions and services for users to use.
```

Comment:
```suggestion
The figure below shows the high-level system architecture for a key node provided by MoveIt called `move_group <https://docs.ros.org/noetic/api/moveit_ros_move_group/html/move__group_8cpp.html>`_. This node serves as an integrator: pulling all the individual components together to provide a set of ROS actions and services for users to use. Note that moveit_commander is `not currently available in MoveIt 2 <https://github.com/ros-planning/moveit2/issues/314>`_.
```

moveit cpp probably didn't exist when this was written

Reply:
Yeah, I think moveitcpp didn't exist back then.